### DISCUSSION TITLE
Will the partial updates wear out the life of E-Paper display?

### DISCUSSION THREAD

**Discussion by 0015** [vroland/epdiy#386] [General]
I made a fridge calendar with EPDIY v7 board and ED097TC2(https://youtu.be/2Iy_9JYkWGs). It uses Google Calendar API to get information about events and update the screen. When I draw the calendar, I don't draw the whole thing at once, but update it in parts one by one. This gives it a bit of an analog feel and I like that kind of stuff, so I did it this way intentionally.

After sharing this project, I heard that partial updates shorten the life of E-Paper. Does this really have anything to do with the life of E-Paper displays?

**Comment by martinberlin** [comment]
No I don’t think that’s true. This displays inside have just some shift registers and write operations on each line, that you later see all together.
No matter if you update 20 lines or the full size, there is no such a thing as full update or partial, everything is treated the same. If it wears out yeah for sure after million operations and being a fluid is possible to leave some permanent ghosts or similar. Nothing last forever and that’s also the case with this displays.
Of course they may last longer if you update it once per hour instead of making hundreds of updates each day but that doesn’t mean that a partial update is worse than a full update. I never read something like that and I think is false
